### ISSUE TITLE
Non chained calls should be on the same line if short enough

### ISSUE THREAD

**Issue by hiqua** [pinterest/ktlint#1077]
## Expected Rule behavior
Right now, ktlint (with `--experimental`) does not complain with this snippet:
```kotlin
class MissingRules {
    fun f() {
        3

            .compareTo(2)
    }
}
```

In this case, since `3.compareTo(2)` is shorter than the maximum line length and it does not consist of chained calls, and it should be written in one line. With long arguments, it should still look like `3.compareTo(` with the arguments on the next lines.

This kind of superfluous / misleading line breaks can happen with automatic conversion from Java to Kotlin using Android Studio.

## Additional information
* Current version of ktlint: 0.40.0
* Styleguide section: I don't think this is covered in the style guides, they mostly mention when the line *should* be broken rather than not.

**Comment by Tapchicoma** [comment]
I am not sure that such rule should exist as such style is subjective - in more complex scenarios some people tend to move `.some_statement()` to the next line.

Not related directly to the topic - KtLint didn't complain about empty line between `3` and `.compareTo(2)`? :thinking:

**Comment by hiqua** [comment]
> KtLint didn't complain about empty line between 3 and .compareTo(2)?

No, when run like this: `ktlint --experimental MissingRules.kt`

Version:
```
ktlint --version
0.40.0
```

**Comment by paul-dingemans** [comment]
> > KtLint didn't complain about empty line between 3 and .compareTo(2)?
> 
> No, when run like this: `ktlint --experimental MissingRules.kt`
> 
> Version:
> 
> ```
> ktlint --version
> 0.40.0
> ```

Fixed in https://github.com/pinterest/ktlint/pull/1105

**Comment by hiqua** [comment]
I'm still unsure about codebases where something like:
```kt
someLongVariable
    .someNonChainedCall()
```
would be normal.

Do you have such an example? I can think of things like
```
("some" + "string" + "which is concatenated making the line very long")
    .someStringMethod()
```

but even then I don't know if people would format it this way unless it breaks the line limit.

In any case free to close if you don't think it makes sense to implement this.

**Comment by Tapchicoma** [comment]
It is very subjective and if you still want to have it - best option for you would be to implement custom rule.
